recipes/libical/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libical" %}
+{% set version = "3.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7ad550c8c49c9b9983658e3ab3e68b1eee2439ec17b169a6b1e6ecb5274e78e6
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage('libical', max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - cmake >=3.11.0
+    - perl
+    - pkg-config  # [osx]
+  host:
+    - icu
+    - glib >=2.32
+    - libxml2 >=2.7.3
+    - gobject-introspection >=0.6.7  # [linux]
+  run:
+    - icu
+    - glib >=2.32
+    - libxml2 >=2.7.3
+    - gobject-introspection >=0.6.7  # [linux]
```

Comment:
Does this need a pinning to minor version?

Reply:
This isn't a hard pin.  This is what the docs say the version constraint is for all of these packages

Reply:
I mean, you should have
```yaml
- {{ pin_compatible("gobject-introspection", max_pin="x.x") }}
```